extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/BeanInitializedBuildItem.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.arc.deployment;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Marker build item to indicate that a build step initializes a CDI bean "manually" through a recorder
```

Comment:
Would it make sense to add a link to `io.quarkus.runtime.annotations.Recorder` so that it's clear what recorder we have in mind?